.idea/codeStyleSettings.xml

```diff
@@ -1,264 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this file deleted?

Reply:
Actually I haven't deleted anything at all, as far as I know.

Reply:
Clearly the diff says differently :)